config/locales/en/settings.yml

```
@@ -240,3 +237,13 @@ en:
       more_info_link_description: Link shown in the consent banner to the cookie policy page.
       version_name: Version name
       version_name_description: Change the version name to force all users to go through the cookie consent process again. This feature can be handy when introducing drastic cookie policy changes requiring a new user acknowledgement.
+    llm:
+      title: LLM Settings
+      provider: "LLM Provider"
+      default_help_text: "Providers will be <strong>disabled</strong> until credentials are configured in the secrets.yml."
```

This key "default_help_text" is no longer used in the component, but the text still makes sense as the setting description.
If we rename it to "provider_description", the row component will automatically display it as the setting description.
Since we usually avoid adding HTML tags in setting descriptions, we could probably remove the `<strong>`  as well. 🤔

```suggestion
      provider_description: "Providers will be disabled until credentials are configured in the secrets.yml."
```

Same in `config/locales/es/settings.yml`